lib/src/null_safety_analysis.dart

```diff
@@ -86,29 +85,38 @@ class NullSafetyAnalysis {
         },
         sources: _systemCache.sources));
 
+    final rootLanguageVersion =
+        (await packageId.source.bind(_systemCache).describe(packageId))
+            .languageVersion;
+    if (!rootLanguageVersion.supportsNullSafety) {
+      return NullSafetyAnalysisResult(
+          NullSafetyCompliance.notCompliant, 'Is not opting in to null safety');
```

Comment:
Can we be more specific? Like "Package $packageId has not opted into null-safety with the environment.sdk constraint in pubspec.yaml", or something like it.